tutorials/arvo/parsing.md

```diff
@@ -268,13 +269,15 @@ themselves. Thus we have several functions for exactly this purpose.
 
 These functions take in either a `tape` or an `@` (typically representing a
 `cord`) with a `rule` and attempts to parse the input with the `rule`. If the
```

Comment:
```suggestion
These functions take in either a `tape` or a `cord`,
alongside a `rule`, and attempt to parse the input with the `rule`. If the
```

I know they technically take a `@`, but doing parsing on atoms that aren't `@t` (or "deeper", `@tas` etc) is unlikely to be sane.